gui/src/app/SPAnalysis/SetupSPAnalysis.tsx

```diff
@@ -0,0 +1,77 @@
+import { FunctionComponent, PropsWithChildren, useMemo, useReducer } from "react"
+import { SPAnalysisContext } from "./SPAnalysisContext"
+
+type SetupSPAnalysisProps = {
+    sourceDataUri: string
+}
+
+type AnalysisFiles = {
+    [key: string]: string
+}
```

Comment:
As noted above, I have renamed this from AnalysisFiles to KVStore to suggest that it is just a key-value store used internally for the implementation. It is left general so it can be replaced by something more specific.